content/blog/2021-12-08-apisix-integrate-skywalking-plugin/apisix-integrate-skywalking-plugin.md

```diff
@@ -0,0 +1,198 @@
+---
+title: "Apache APISIX Integrates with SkyWalking to Create a Full Fange of Log Processing"
+author: "Haochao Zhuang"
```

Comment:
I can see this is @dmsolr. Are you co-author of this blog? It is rarely to see submitting a pull request on behalf of another individual. Could @dmsolr or @lijing-21 confirm what is the status?

Reply:
Yes, the author of the article is @dmsolr. Now I am in charge of issuing the cooperation invitations to other communities for Apache APISIX community, so I submit this pull request for him. And the author himself is aware of this.

Reply:
@dmsolr I need your confirmation and review to make sure this doesn't violate your IP and willings.

Reply:
Thank you for helping me to synchronize the blog here, @lijing-21 
I already had known and agreed before. Thanks for your remind, @wu-sheng